OpenEdXMobile/res/layout/activity_register.xml

```diff
@@ -3,120 +3,135 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@android:color/white"
     android:orientation="vertical"
-    android:paddingBottom="8dp"
-    android:background="@android:color/white">
+    android:paddingBottom="8dp">
```

Comment:
Thats not logically correct, we're assigning an input padding to a ViewGroup which might or might not be an input based view.

You can just define a new `dimen`.